.github/workflows/test_export_onnx.yml

```diff
@@ -31,7 +31,7 @@ jobs:
     - name: Test with unittest
       working-directory: tests
       run: |
-        pytest exporters/onnx/test_onnx_*.py -s -n auto -m "not tensorflow_test" --durations=0
+        pytest exporters/onnx/test_onnx_*.py -s -n auto -m "not tensorflow_test and not timm_test" --durations=0
```

Comment:
Can you add `timm_test` to https://github.com/huggingface/optimum/blob/fb57f9f01e9cf8bfcf4eb02ecf9beefd6b47f8b3/pyproject.toml#L33 ?